sympy/rubi/utility_function.py

```diff
@@ -65,3 +67,101 @@ def Sqrt(a):
 
 def ArcCosh(a):
     return acosh(a)
+
+def TogetherSimplify(expr):
+    return
+
+def Coefficient(u, var, n):
+    return
+
+def RemoveContent():
+    return
+
+def ExpandIntegrand(expr, x):
+    return
+
+def ArcCosh(a):
+    return acosh(a)
+
+def With():
+    return
+
+def Denominator(var):
+    return fraction(var, exact=True)[1]
+
+def Hypergeometric2F1(a, b, c, z):
+    return hyp2f1(a, b, c, z)
+
+def TogetherSimplify():
+    return
+
+def IntLinearcQ():
+    return
+
+def ArcTan(a):
+    return atan(a)
+
+def Not(var):
+    return not(var)
+
+def Simplify():
+    return
+
+def FractionalPart(a):
+    return FracPart(a)
+
+def IntegerPart(a):
+    return IntPart(a)
+
+def Simp():
+    return
+
+def Rt():
+    return
+
+def SumSimplerQ():
+    return
+
+# utility functions used in tests
+
+def AppellF1(a, b1, b2, c, x, y):
+    return appellF1(a, b1, b2, c, x, y)
+
+def Integrate(f, x):
+    return integrate(f, x)
+
+def hypergeom():
+    return
+
+def EllipticPi(*args):
+    return ellippi(*args)
+
+def EllipticE(*args):
+    return ellipe(*args)
+
+def EllipticF(Phi, m):
+    return ellipf(Phi, m)
+
+def arctan(a):
+    return atan(a)
+
+def arctanh(a):
+    return atanh(a)
+
+def arcsin(a):
+    return asin(a)
+
+def arcsinh(a):
+    return asinh(a)
+
+def arccos(a):
```

Comment:
ooh, the tests we took are from Maple. not mathematica. Dont worry about implementing them now.